test/geometries/multi_linestring.cpp

```diff
@@ -0,0 +1,147 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+// Unit Test
+
+// Copyright (c) 2020 Digvijay Janartha, Hamirpur, India.
+
+// Parts of Boost.Geometry are redesigned from Geodan's Geographic Library
```

Comment:
Is this needed for completely new files?

Reply:
You mean line 6, shall I remove this in all the files?

Reply:
Good questions. I think the canonical form of the notice is just

```
// Boost.Geometry (aka GGL, Generic Geometry Library)
//
// Copyright (c) 2020 Digvijay Janartha, Hamirpur, India.
//
// Use, modification and distribution is subject to the Boost Software License,
// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
// http://www.boost.org/LICENSE_1_0.txt)
```

The other copyrights are added as necessary by authors of further modifications, that is according to [How should Boost programmers maintain the copyright messages?](https://www.boost.org/users/license.html).

Reply:
Ok I'll change to this form in the next commit, alongside tests for other cs as well :)